internal/system/agent/router.go

```diff
@@ -33,17 +32,17 @@ func LoadRestRoutes() *mux.Router {
 	r := mux.NewRouter()
 	b := r.PathPrefix("/api/v1").Subrouter()
 
-	b.HandleFunc("/"+OPERATION, operationHandler).Methods(http.MethodPost)
-	b.HandleFunc("/"+CONFIG+"/{"+SERVICES+"}", configHandler).Methods(http.MethodGet)
-	b.HandleFunc("/"+METRICS+"/{"+SERVICES+"}", metricsHandler).Methods(http.MethodGet)
+	b.HandleFunc("/"+operation, operationHandler).Methods(http.MethodPost)
```

Comment:
Do either of you have strong feelings about this?

@tsconn23 @anonymouse64

Reply:
I do not have strong feelings about it, I like upper case constants to make it clear it's a constant and not a local variable, but I do think trying to match idiomatic Go is good.
I'm not concerned about exporting this from the package, since these are all in the `internal/` packages anyways, so no one outside of edgex-go will ever be able to consume these.